src/components/controls/textbox/Textbox.js

```diff
@@ -91,13 +92,17 @@ class Textbox extends React.Component {
     super(props);
 
     this.state = {
-      currentValue: ''
+      currentValue: props.value
```

Comment:
Not setting this value to an initial value results in console warnings regarding controlled vs. uncontrolled inputs.